app/controllers/organizations/staff/staff_controller.rb

```diff
@@ -1,43 +1,32 @@
 class Organizations::Staff::StaffController < Organizations::BaseController
-  before_action :set_staff_account, only: [:deactivate, :activate, :update_activation]
+  before_action :set_staff, only: [:update_activation]
 
   layout "dashboard"
 
   def index
-    authorize! StaffAccount, context: {organization: Current.organization}
+    authorize! User, context: {organization: Current.organization}
 
-    @staff_accounts = authorized_scope(StaffAccount.all)
+    @staff = authorized_scope(User.staff)
   end
 
-  def deactivate
-    @staff_account.deactivate
-    respond_to do |format|
-      format.html { redirect_to staff_staff_index_path, notice: t(".success") }
-      format.turbo_stream { render "organizations/staff/staff/update" }
+  def update_activation
+    if @staff.deactivated_at
+      @staff.activate
```

Comment:
Not directly related to your code changes, but when calling method to update a record I think it should be a bang `!` method i.e., `activate!` and `deactivate!`. Thoughts? We can address in a new issue to keep this PR on scope, if you prefer.

Reply:
That's not the typical bang naming convention, and since that convention is fairly established think it is best we not mess with it.

Reply:
Ah, I see that you wanted to change these to use `update!` so understand this idea better.

To me, seeing a bang method indicates that there is a non-bang version. I don't think methods typically have bangs unless a safer version exists too.

Reply:
Interesting, I had not thought of it that way. I learned it was to signal a dangerous method that will modify the object permanently, and also will fail loudly. It’s a way to let developers know it’s no ordinary instance method.

This thread is an interesting discussion on it that echoes that, but also some people mention what you state, including Matz by the looks of it. https://stackoverflow.com/questions/612189/why-are-exclamation-marks-used-in-ruby-methods

What’s a good way to let the developer know that calling a particular instance method will actually update the record, if we don’t use a bang? A bang is the convention used where I work…but I am open to ideals and not chained to it.

Reply:
If Matz made a statement on it, I'm inclined to follow his guidance. There are plenty of mutating methods in Ruby that don't have bangs in them, as I see that thread also brings up. Could go into further debate, but ultimately this is dev preferences which are hard to change. We can discuss this more if we think it needs to be in the next meeting to come to a project consensus.

In regards to your question, I hope devs aren't just calling methods without knowing what they do, purely on naming conventions! I don't think Ruby is often written functionally, so I would expect devs to expect something called "activate" to mutate the object.

Reply:
To clarify...I agree that devs should not be calling methods wihtout knowing what they do haha...but the `!` is an instant communicator it's a mutating method. But, I am fine with this implementation as well.